float32 chassis_arm1_joint
float32 arm1_arm2_joint
float32 arm2_arm3_joint 
float32 arm3_arm4_joint
